string-manipulation-1/src/is-vowel.js

```diff
@@ -0,0 +1,10 @@
+/* exported isVowel */
+// pseudocode
+// List out vowels A, E, I, O, U, a, e, i, o, u. You must include both capital and lowercase letters.
+// Create storage for the above list.
+// If the character provided includes one of these letters in the storage from above then return true.
+// All other letters should return false.
+var vowels = ['A', 'E', 'I', 'O', 'U', 'a', 'e', 'i', 'o', 'u'];
```

Comment:
You should move this variable inside the function

Reply:
Thanks! Any advantages of having this variable inside the function as opposed to outside?

Reply:
it reduces how long the variable exists and is taking up memory. In general, a variable should only exist as long as is necessary